back-end/services/checkReasons.service.js

```diff
@@ -0,0 +1,93 @@
+export const PRIVACY_KEYWORDS = ["provide", "email", "phone", "mobile number",
```

Comment:
Is "provide" exactly a keyword for privacy concerns? For example, "please provide your name" wouldn't necessarily be a concern. I think sticking to nouns would be a good call!

Reply:
Also, I wonder if there's a bot or something we could import that automatically gives us privacy keywords. Though, this goes beyond our MVP. Maybe we could consider this if we have time later :)

Reply:
some more keywords:
- phone number (we currently just have "phone")
- address
- SIN / Social Security Number

Reply:
Something to think about: Asking for things like just a name or just a location doesn't seem like much of a privacy concern. But when all of these are connected (e.g., the bot asks for your name, then location, then your occupation, ...), it starts to build up a concern. The user may leave the chatbot after a specific number of PII requests. How could we analyze this?

Reply:
If you look over to how the function actually analyses the privacy keywords you'll see it doesn't return true the first time the keyword if found but goes from the last dialogue tll half of the transcript and if a keyword if found, it checks if the customer replied with "no", "not comfortable", etc to see if it was actually a privacy concern